code/modules/mob/living/carbon/xenomorph/castes/Boiler.dm

```diff
@@ -61,6 +61,7 @@
 	weed_food_states_flipped = list("Boiler_1","Boiler_2","Boiler_3")
 
 	var/datum/effect_system/smoke_spread/xeno_acid/smoke
+	var/last_psychic_pulse_time = 0
```

Comment:
add a comment for what this variable is for e.g. 

```suggestion
    /// Last time a dropship camera detection pulse was triggered
	var/last_psychic_pulse_time = 0
```

consider using COOLDOWN_DECLARE (basically just a pretty formatting for this kind of world.time cooldown timer.)